### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by weironghuang31**

### CODE REVIEW

**Comment by wadehuang36** [comment]
merge

**Review by uglycitrus — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ allprojects {
 
 3.  In your app level build.gradle files include the following lines:
 ```groovy
-compile(name: 'avenue-analytics', ext: 'aar') {
-  transitive = true
```
Comment (somya) [inline README.md:21]: what does transitive = true do? Why was it removed?

File: README.md [hunk 2]
```diff
@@ -17,14 +17,16 @@ allprojects {
...
-}
+compile(name: 'avenue-analytics', ext: 'aar')
+
 compile 'com.google.code.gson:gson:2.8.0'
-compile 'com.google.android.gms:play-services-basement:11.0.4'
-compile 'com.google.android.gms:play-services-analytics:11.0.4'
+
+// if you use the library before 2017-11-01, you have to add this dependency, too.
+compile 'com.google.android.gms:play-services-basement:10.0.0'
```
Comment (somya) [inline README.md]: Can you explain a little more. Also you're rolling back the version to 10.0.0?

**Comment by weironghuang31** [comment]
@uglycitrus can you merge this pull request?

**Pull request merged**